programmers/음양-더하기-for.js

```diff
@@ -0,0 +1,9 @@
+function solution(absolutes, signs) {
+    let answer = 0;
+    for (let i = 0; i < signs.length; i++) {
+        if (signs[i] === false) absolutes[i] = -absolutes[i];
+    }
+    
+    for (let absol of absolutes) answer += absol   
```

Comment:
absolutes 의 원소는 absolute 로 짓는것이 더욱 적절하다.
```suggestion
    for (let absolute of absolutes) answer += absolute   
```